sparkly/writer.py

```diff
@@ -196,6 +197,52 @@ def mysql(self, host, database, table, port=None, mode=None, parallelism=None, o
 
         return self._basic_write(writer_options, options, parallelism, mode)
 
+    def kafka(self,
+              host,
+              topic,
+              key_serializer,
+              value_serializer,
+              port=None,
+              parallelism=None,
+              options=None):
+        """Writes dataframge to kafka topic.
+
+        Expected schema is:
+          key -> Struct(...)
+          value -> Struct(...)
+
+        Args:
+            host (str): Kafka host.
+            topic (str): Topic to write to.
+            key_serializer (function): Function to serialize key.
+            value_serializer (function): Function to serialize value.
+            port (int|None): Kafka port.
+            parallelism (int|None): The max number of parallel tasks that could be executed
+                during the write stage (see :ref:`controlling-the-load`).
+            options (dict|None): Additional options.
+        """
+        def write_partition_to_kafka(messages):
+            producer = KafkaProducer(
+                bootstrap_servers=['{}:{}'.format(host, port or 9092)],
+                key_serializer=key_serializer,
```

Comment:
can we describe in the docs how serialiser/deserialiser functions look like?
is it: `bytes -> Row -> bytes` mappers or `bytes -> dict -> bytes`?